docs/Inflight Adjustments.md

```diff
@@ -113,45 +113,45 @@ this reason ensure that you define enough ranges to cover the range channel's us
 | 8     | PITCH_ROLL_D |
 | 9     | YAW_P |
 | 10    | YAW_I |
-| 11    | YAW_D |
+| 11    | YAW_D_FF |
 | 12    | RATE_PROFILE | Switch between 3 rate profiles using a 3 position switch. |
 | 13    | PITCH_RATE |
 | 14    | ROLL_RATE |
 | 15    | PITCH_P |
 | 16    | PITCH_I |
-| 17    | PITCH_D |
+| 17    | PITCH_D_FF |
 | 18    | ROLL_P |
 | 19    | ROLL_I |
-| 20    | RC_YAW_EXPO |
-| 21    | MANUAL_RC_EXPO |
-| 22    | MANUAL_RC_YAW_EXPO |
-| 23    | MANUAL_PITCH_ROLL_RATE |
-| 24    | MANUAL_ROLL_RATE |
-| 25    | MANUAL_PITCH_RATE |
-| 26    | MANUAL_YAW_RATE |
-| 27    | NAV_FW_CRUISE_THROTTLE |
-| 28    | NAV_FW_PITCH2THR |
-| 29    | ROLL_BOARD_ALIGNMENT |
-| 30    | PITCH_BOARD_ALIGNMENT |
-| 31    | LEVEL_P |
-| 32    | LEVEL_I |
-| 33    | LEVEL_D |
-| 34    | POS_XY_P |
-| 35    | POS_XY_I |
-| 36    | POS_XY_D |
-| 37    | POS_Z_P |
-| 38    | POS_Z_I |
-| 39    | POS_Z_D |
-| 40    | HEADING_P |
-| 41    | VEL_XY_P |
-| 42    | VEL_XY_I |
-| 43    | VEL_XY_D |
-| 44    | VEL_Z_P |
+| 20    | ROLL_D_FF |
+| 21    | RC_YAW_EXPO |
+| 22    | MANUAL_RC_EXPO |
+| 23    | MANUAL_RC_YAW_EXPO |
+| 24    | MANUAL_PITCH_ROLL_RATE |
+| 25    | MANUAL_ROLL_RATE |
+| 26    | MANUAL_PITCH_RATE |
+| 27    | MANUAL_YAW_RATE |
+| 28    | NAV_FW_CRUISE_THROTTLE |
+| 29    | NAV_FW_PITCH2THR |
+| 30    | ROLL_BOARD_ALIGNMENT |
+| 31    | PITCH_BOARD_ALIGNMENT |
+| 32    | LEVEL_P |
+| 33    | LEVEL_I |
+| 34    | LEVEL_D |
+| 35    | POS_XY_P |
+| 36    | POS_XY_I |
+| 37    | POS_XY_D |
+| 38    | POS_Z_P |
+| 39    | POS_Z_I |
+| 40    | POS_Z_D |
+| 41    | HEADING_P |
+| 42    | VEL_XY_P |
+| 43    | VEL_XY_I |
+| 44    | VEL_XY_D |
 | 45    | VEL_Z_P |
 | 46    | VEL_Z_I |
 | 47    | VEL_Z_D |
 | 48    | FW_MIN_THROTTLE_DOWN_PITCH_ANGLE |
-| 49    | PROFILE | Switch between 3 rate profiles using a 3 position switch. |
```

Comment:
@Scavanger was this intended?

Reply:
It's just a fix in the documentation, rate profile is currently inactive in the code.